containers/R_Python/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM clipper/py-rpc:latest
```

Comment:
Add a line to the [`bin/build_docker_images.sh`](https://github.com/ucbrise/clipper/blob/develop/bin/build_docker_images.sh) script to build this container